parts/language/lang_en.lua

```diff
@@ -934,7 +934,7 @@ return{
         "You can connect a keyboard to your phone or tablet!",
         "You can navigate the menu with a keyboard, but only in this screen.",
         "You can open the save directory from the Stats page.",
-        "You can perform a spin with 28 of the 29 minoes in this game; the exception being O1.",
+        "You can perform a spin with 28 of the 29 minos in this game; the exception being O1.",
```

Comment:
Someone please fact check this because it seems really inconsistent: Tetrimino**s**, tetromino**es**, mino... s? es?

Reply:
There's no standard, we just have to be consistent.